app/client/src/components/formControls/DynamicTextFieldControl.tsx

```diff
@@ -65,7 +65,7 @@ class DynamicTextControl extends BaseControl<
 
     return (
       <Wrapper
-        className={`t--${configProperty}`}
+        className={`t--${configProperty} dynamic-text-feild-control`}
```

Comment:
```suggestion
        className={`t--${configProperty} dynamic-text-field-control`}
```

Spelling issue in "field"